engine/src/main/java/org/hibernate/search/store/impl/FSMasterDirectoryProvider.java

```diff
@@ -69,12 +71,12 @@ public void initialize(String directoryProviderName, Properties properties, Buil
 		this.properties = properties;
 		this.directoryProviderName = directoryProviderName;
 		//source guessing
-		sourceDir = DirectoryProviderHelper.getSourceDirectory( directoryProviderName, properties, true );
-		log.debugf( "Source directory: %s", sourceDir.getPath() );
-		indexDir = DirectoryHelper.getVerifiedIndexDir( directoryProviderName, properties, true );
-		log.debugf( "Index directory: %s", indexDir.getPath() );
+		sourceDir = DirectoryProviderHelper.getSourceDirectoryPath( directoryProviderName, properties, true );
+		log.debugf( "Source directory: %s", sourceDir );
+		indexDir = DirectoryHelper.getVerifiedIndexPath( directoryProviderName, properties, true );
```

Comment:
You may want to normalize the path at some point (`indexDir = indexDir.normalize()`), since previously we used `indexDir.getCanonicalPath()` to set `indexName`. Or maybe it's not important now that we don't use it for `equals` anymore?